addons/tmi/tmi.gd

```diff
@@ -17,6 +17,7 @@ class EventType:
 
 var _load_stack = {}
 var _enrichable = []
+var _prev_connection_state = ConnectionStatus.NOT_CONNECTED
```

Comment:
this shouldn't be needed as an instance variable of tmi because it's a state tracker solely for the poller.

Reply:
Can you confirm that the poller is correctly tracking the prev_connection_state? On my end, it's emitting connection_status_changed on every timeout because it believes it's still switching from NOT_CONNECTED to EVENTSUB. 

I did make a typo below as _prev_connection_state shouldn't also be a local variable.